standalone/content/js/standaloneRoomViews.jsx

```diff
@@ -788,6 +802,84 @@ loop.standaloneRoomViews = (function(mozL10n) {
     }
   });
 
+  var Slide = React.createClass({
+    propTypes: {
+      imageClass: React.PropTypes.string.isRequired,
+      indexClass: React.PropTypes.string.isRequired,
+      joinRoom: React.PropTypes.func.isRequired,
+      text: React.PropTypes.string.isRequired,
+      title: React.PropTypes.string.isRequired
+    },
+
+    handleGotItClick: function(event) {
```

Comment:
Please move this function to the component containing the overlay element and pass it in as a callback.  The callback (in addition to the stuff in this function) simply should change this.state (or dispatch a flux action if that's appropriate) so that it re-renders without the overlay element.